app/controllers/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def index
+    @users = User.all
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to @user
+    else
+      render 'edit'
+    end
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def create
```

Comment:
You have a view for this action but this action doesn't need a view so, please, remove it.

Reply:
Removed on next commit!